dessia_common/graph.py

```diff
@@ -56,3 +56,43 @@ def extract_region(networkx_graph: nx.Graph, nodes, distance: int = 5):
         region_nodes.extend(ancs)
 
     return networkx_graph.subgraph(region_nodes)
+
+
+def get_longest_path(graph: nx.DiGraph, begin, end) -> list:
+    return list(nx.shortest_simple_paths(graph, begin, end))[-1]
+
+
+def get_paths_from_to(graph: nx.DiGraph, _from: list, _to: list) -> list:
```

Comment:
We have to be careful with types here. `list` is ambiguous over python version and should not be used (use `List`, instead). In addition to that, it should never be used without an argument (`List[SomeType]`), otherwise it is subject to errors. 

In any case, this is not mandatory as no jsonschema would be created from this, so I even prefer no typing (and thus no weird and unwanted behavior) over incomplete ones.

This goes for other parts of the code (get longest_path,...) and Dicts as well (get_distance_by_nodes,...)